array-manipulation-2/src/take.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
+function take(array,count){
+    let newArr = [];
+    for(let i = 0;i<count;i++){
+        if(array[i]===undefined){
+            newArr = [];
+            return newArr
+        }

```

Comment:
🤔 this part is kind of confusing. you can check more than one condition in your `for` loop

```js
for (let i = 0; i < count && i < array.length; i++) {
```

that way you don't need to have this `if` in here.